core/tests/protractor_utils/forms.js

```diff
@@ -185,23 +190,22 @@ var RichTextEditor = async function(elem) {
   var closeRteComponentButtonLocator = by.css(
     '.protractor-test-close-rich-text-component-editor');
   // Set focus in the RTE.
-  await waitFor.elementToBeClickable(rteElements.first());
-  await (await rteElements.first()).click();
+  var rteElementFirst = await rteElements.first();
+  await action.click('rte Element First', rteElementFirst);
```

Comment:
Can the first argument be changed to `'First RTE Element'`?

Reply:
done!